writing-gleam/command-line-reference.md

```diff
@@ -1,267 +0,0 @@
----
```

Comment:
Was this file intentionally removed?

Reply:
nope, no idea why it is removed, I will restore it